src/android/io/branch/BranchSDK.java

```diff
@@ -887,6 +883,9 @@ public void onRegisterViewFinished(boolean registered, BranchError error) {
             Log.d(LCAT, "RegisterViewStatusListener registerViewFinished()");
 
             if (error == null) {
+                // HURDLR: note that this is inconsistent with iOS, but we don't use it yet.
```

Comment:
the listener in iOS doesn't have a boolean loggedOut we can use, so there is an API discrepancy between iOS & Java here